src/common/exif.cc

```diff
@@ -205,8 +205,18 @@ static void dt_remove_exif_keys(Exiv2::ExifData &exif, const char *keys[], unsig
 {
   for(unsigned int i = 0; i < n_keys; i++)
   {
-    Exiv2::ExifData::iterator pos = exif.findKey(Exiv2::ExifKey(keys[i]));
-    if(pos != exif.end()) exif.erase(pos);
+    try
+    {
+      Exiv2::ExifData::iterator pos = exif.findKey(Exiv2::ExifKey(keys[i]));
```

Comment:
Exif can have duplicate tags, i.e. multiple tags with the same name but different values.
So this should be more like:
```
Exiv2::ExifData::iterator pos;
while((pos = exif.findKey(Exiv2::ExifKey(keys[i]))) != exif.end()) exif.erase(pos);
```
No, i'm not just picking on the changes, the changes expose faults and while there, i'd like to get rid of the most obvious-ones.